src/js/views/programs/sidebar/action/action-sidebar.hbs

```diff
@@ -17,7 +17,8 @@
     <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.programs.sidebar.action.actionSidebarTemplate.ownerLabel }}</h4><div class="flex-grow" data-owner-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.programs.sidebar.action.actionSidebarTemplate.dueDayLabel }}</h4><div class="flex-grow" data-due-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.programs.sidebar.action.actionSidebarTemplate.formLabel }}</h4><div class="flex-grow" data-form-region></div></div>
-    <div class="flex u-margin--t-24" data-form-sharing-region></div>
+    <div class="u-margin--t-24" data-form-sharing-region></div>
+    <div class="u-margin--t-24" data-allow-uploads-region></div>
```

Comment:
I removed `flex` from the `data-form-sharing-region` region as well.

Everything looks the same without it.